tests/index.test.ts

```diff
@@ -159,6 +159,37 @@ describe('Toolkit', () => {
     })
   })
 
+  describe('.run', () => {
+    it('runs the async function passed to it', async () => {
```

Comment:
Could we use a `jest.fn()` mock function here? Or is there some nuance to testing the async function that I'm not picking up on? Like:

```js
const fn = jest.fn()
await Toolkit.run(fn)
expect(fn).toHaveBeenCalled()
expect(fn.mock.calls[0][0]).toBeInstanceOf(Toolkit)
```